library/FrontRow/App/Database.hs

```diff
@@ -29,22 +29,30 @@ import Control.Monad.Logger (runNoLoggingT)
 import Control.Monad.Reader
 import Data.ByteString (ByteString)
 import qualified Data.ByteString.Char8 as BS8
+import Data.Foldable (for_)
 import Data.IORef
 import Data.Pool
 import qualified Data.Text as T
 import Data.Time (UTCTime, getCurrentTime)
 import Database.Persist.Postgresql
+import Database.Persist.Sql.Raw.QQ (executeQQ)
 import Database.PostgreSQL.Simple (connectPostgreSQL)
 import qualified FrontRow.App.Env as Env
 import System.Process (readProcess)
 
 type SqlPool = Pool SqlBackend
 
+newtype Seconds = Seconds { getSeconds :: Int }
```

Comment:
You'll need to export this, so users can define `getStatementTimeout`. I would probably put it in `FrontRow.App.Seconds` or `FrontRow.App.Time` myself, but exporting it from here seems fine to start too.

Reply:
We also have a strong `un`-convention over `get`, for newtypes, FWIW.

```hs
newtype Seconds = Seconds
  { unSeconds :: Int
  }
```